spec/requests/users_request_spec.rb

```diff
@@ -25,19 +25,19 @@ def create_valid_user
   end
 
   it 'show action returns the requested user when passed a valid token' do
-    get '/users', headers: authenticated_token
+    get '/user', headers: authenticated_token
```

Comment:
Resources are suppose to be plural as far as i am aware. That's why rails does `resources **bridges**`